flaskbb/utils/helpers.py

```diff
@@ -759,3 +767,24 @@ def wrapper(*a, **k):
 def register_view(bp_or_app, routes, view_func, *args, **kwargs):
     for route in routes:
         bp_or_app.add_url_rule(route, view_func=view_func, *args, **kwargs)
+
+
+class FlashAndRedirect(object):
```

Comment:
Needed a "thunkable" way to cause a redirect, if you look through the commits there used to be a pair of functions called `flash_and_redirect` and `thunk` that were actually a less useful version of this class.